docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+  <finalization candidates="195" enqueued="2" />
+  <ownableSynchronizers candidates="2089" cleared="0" />
+  <references type="soft" candidates="3059" cleared="0" enqueued="0" dynamicThreshold="0" maxThreshold="32" />
+  <references type="weak" candidates="10797" cleared="0" enqueued="0" />
+  <references type="phantom" candidates="6" cleared="0" enqueued="0" />
+  <stringconstants candidates="10031" cleared="0"  />
+</gc-op>
+<gc-op id="2017" type="classunload" timems="0.123" contextid="2003" timestamp="2021-03-05T12:16:43.468">
+  <classunload-info classloadercandidates="25" classloadersunloaded="0" classesunloaded="0" anonymousclassesunloaded="0" quiescems="0.000" setupms="0.123" scanms="0.000" postms="0.000" />
+</gc-op>
+<gc-op id="2018" type="sweep" timems="5.120" contextid="2003" timestamp="2021-03-05T12:16:43.474" />
+<gc-op id="2019" type="compact" timems="762.323" contextid="2003" timestamp="2021-03-05T12:16:44.236">
+  <compact-info movecount="8024461" movebytes="163375400" />
+  <remembered-set-cleared processed="777104" cleared="777104" durationms="2.188" />
+</gc-op>
+```
+
+The global cycle's increment ends. The end of the increment is recorded with `<gc-end>` and provides another snapshot of memory allocation on the heap, similar to `<gc-start>`.
+
+
+```xml
+<gc-end id="2020" type="global garbage collect" contextid="2003" durationms="1126.788" usertimems="7971.788" systemtimems="1.000" stalltimems="1016.256" timestamp="2021-03-05T12:16:44.237" activeThreads="8">
+  <mem-info id="2021" free="1572864" total="838860800" percent="0">
+    <mem type="eden" free="1572864" total="1572864" percent="100" />
+    <pending-finalizers system="2" default="0" reference="0" classloader="0" />
+    <remembered-set count="874496" freebytes="29884416" totalbytes="33382400" percent="89" regionsoverflowed="0" regionsstable="0" regionsrebuilding="0"/>
+  </mem-info>
+</gc-end>
+```
+
+Comparing the snapshot at the beginning and end of this STW `global mark phase` subincrement shows that memory has been reclaimed and regions reassigned to create an empty eden space, equal to 1.5MB(1,572,864 B). In some cases, a global cycle also reclaims a small amount of memory from non-eden regions.
+
+The cycle ends (`<cycle-end>`). The following `<allocation-satisfied>` element indicates that the allocation request that caused the allocation failure can now complete successfully.
+
+```xml
+<cycle-end id="2022" type="global garbage collect" contextid="2003" timestamp="2021-03-05T12:16:44.237" />
+<allocation-satisfied id="2023" threadId="00000000008A9E00" bytesRequested="24" />
+<af-end id="2024" timestamp="2021-03-05T12:16:44.237" threadId="00000000008AA780" success="true" />
+```
+
+The STW pause ends with the `<exclusive-end>` element.
+
+```xml
+<exclusive-end id="2025" timestamp="2021-03-05T12:16:44.237" durationms="1130.358" />
+```
+
+<!---
+NOW SHOW A GLOBAL CYCLE THAT IS TRIGGERED WHEN NO GLOBAL MARK CYCLE IS RUNNING? (this id="2062" example)
```

Comment:
I would not bother about details. Just show the start (af-start and cycle-start and perhaps gc-start)